velox/exec/Window.cpp

```diff
@@ -100,13 +114,16 @@ Window::WindowFrame Window::createWindowFrame(
           std::dynamic_pointer_cast<const core::ConstantTypedExpr>(frame)
               ->value();
       VELOX_CHECK(!constant.isNull(), "k in frame bounds must not be null");
-      VELOX_USER_CHECK_GE(
-          constant.value<int64_t>(), 1, "k in frame bounds must be at least 1");
-      return std::make_optional(FrameChannelArg{
-          kConstantChannel, nullptr, constant.value<int64_t>()});
+      auto value = VariantConverter::convert(constant, TypeKind::BIGINT)
```

Comment:
We should avoid using variant in production code. Would be nice to refactor in a follow-up.